tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -114,14 +114,68 @@ describe(__filename, () => {
   };
 
   describe('<AddonReviewList/>', () => {
-    it('shows placeholders for reviews without an addon', () => {
-      const root = render({ addon: null });
+    it('displays 4 placeholders when there are no reviews yet', () => {
```

Comment:
It is more than just "no reviews". It is only because `test_count` is set to `0`.

I believe the previous test case with `addon: null` is also valid, no?

Reply:
You're right - this guy is for an existing addon with no reviews whereas the former test is for a null addon.. gotta update descriptions